### PR TITLE
Update Readme

### LINKED ISSUE #173: Update readme
better info on set up
- [ ] getting ruby from rbenv
- [x] set env variables

### PR DESCRIPTION
**Pull request by dlouise64**
close #173

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ This is a ruby on rails app. You will need to:
 git clone git@github.com:ServiceInnovationLab/RapuTure.git
 ```
 
+You will need to setup environment variables
+* Rename `env-example` file to `.env`
```
Comment (Br3nda) [inline README.md]:
```suggestion
> mv env-example .env
```

File: README.md [hunk 2]
```diff
@@ -49,6 +49,23 @@ This is a ruby on rails app. You will need to:
...
+```
+ruby -v
+```
+
+Choose one of these tools, to change the ruby version
+1. [rbenv](https://github.com/rbenv/rbenv)
+2. [rvm](https://rvm.io/)
+
+
+* You will need to setup the apps environment variables
```
Comment (JacOng17) [inline README.md]: You will need to setup the **app's**

File: README.md [hunk 3]
```diff
@@ -106,7 +123,7 @@ gem install bundler
 ```
 bin/setup
 ```
-* Rename `env-example` file to `.env`
+* `mv env-example .env`
```
Comment (JacOng17) [inline README.md]:
```
mv env-example .env
```

File: README.md (hunk 2 above)
Comment (Br3nda) [reply README.md]:
```suggestion
		* You will need to setup the app's environment variables
```

**Pull request merged**